index.html

```diff
@@ -18,16 +18,13 @@
         <div class="topnav">
             <a href="#home" class="active">Welcome</a>
             <div id="myLinks">
-                <a href="#Portfolio">Portfolio</a>
-                <a href="#about">About</a>
-                <a href="#contact">Contact</a>
+                <a class="mobile-menu-option" href="#Portfolio">Portfolio</a>
+                <a class="mobile-menu-option" href="#about">About</a>
+                <a class="mobile-menu-option" href="#contact">Contact</a>
             </div>
             <a href="javascript:void(0);" class="icon" onclick="myFunction()">
                 <i class="fa fa-bars"></i>
             </a>
```

Comment:
- [ ] Your mobile menu items are not aligned, as can be seen in the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) Kindly ensure that your mobile menu items are aligned to the left, similar to the example in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)
- [ ] Kindly ensure that the mobile menu overlay covers the entire mobile screen, including the logo.

**Your design:**        |  **The Figma design:**
:-------------------------:|:-------------------------:
<img width="800" alt="Your desgin" src="https://user-images.githubusercontent.com/12745474/208511329-63c4086a-e957-4404-8cff-d9f327a0e86f.png"> |  <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/208512703-37b15324-bab9-41de-9d64-942b0e083b52.png">